server/mediaserver.go

```diff
@@ -260,19 +260,6 @@ func gotRTMPStreamHandler(s *LivepeerServer) func(url *url.URL, rtmpStrm stream.
 		var sess *BroadcastSession
 
 		if s.LivepeerNode.Eth != nil {
-
-			//Check if round is initialized
-			initialized, err := s.LivepeerNode.Eth.CurrentRoundInitialized()
```

Comment:
👏

Reply:
Why does round no longer need to be initialized?

Reply:
In the V1 protocol, there was a restriction that on-chain jobs could not be created if the current round was not initialized. With the removal of on-chain jobs, there is no longer an initialized round requirement for the broadcaster.

Note: Given our current planned approach of using round initialization to store block hashes which are attached to tickets to prove that a creation round number existed at the time of ticket creation, we will eventually have some dependence on round initialization. However, the current thought is that a broadcaster still does not have to be blocked by round initialization when sending segments to an orchestrator - it can use the block hash of the last initialized round. In this scenario, the broadcaster can always create tickets/segments regardless of whether the current round is initialized. However, the effective validity period of ticket is reduced if the previous round is used instead of the current round. An orchestrator *might* have an incentive to initialize the round in this case so that the broadcaster can use the current round as the creation round to extend the validity period of tickets. We'll ultimately address this in SS, but I think for the time being eliminating this check suffices for MVS.

Reply:
Makes sense, thanks @yondonfu! I see how "extending" the effective validity period by initializing the round would be a strong incentive to do so.